### PR TITLE
Add new designs banner

### PR DESCRIPTION
**Pull request by vanitabarrett**

### CODE REVIEW

File: pages/[categoryId]/index.tsx [hunk 1]
```diff
@@ -18,6 +19,9 @@ export default function CategoryHomepage({
       </Head>
       <h1 className="page-title">All {categoryInfo.name}</h1>
       <CategoryNav categoryInfo={categoryInfo} />
+      <Notice title="Can't find what you're looking for?">
+        <p>New designs added at the start of each month, follow us on <a href="https://www.instagram.com/roseatecards/" rel="noreferrer external">social media</a> for updates!</p>
```
Comment (andymsuk) [inline pages/[categoryId]/index.tsx]:
```suggestion
        <p>New designs are added at the start of each month, follow us on <a href="https://www.instagram.com/roseatecards/" rel="noreferrer external">social media</a> for updates!</p>
```

File: pages/[categoryId]/index.tsx [hunk 2]
```diff
@@ -18,6 +19,9 @@ export default function CategoryHomepage({
       </Head>
       <h1 className="page-title">All {categoryInfo.name}</h1>
       <CategoryNav categoryInfo={categoryInfo} />
+      <Notice title="Can't find what you're looking for?">
```
Comment (andymsuk) [inline pages/[categoryId]/index.tsx:23]: Might this look nicer at the bottom of the page?

File: styles/components/_notice.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
+.notice {
+  background-color: $focus;
+  width: 90%;
```
Comment (andymsuk) [inline styles/components/_notice.scss:3]:
```suggestion
  width: 100%;
```
Might look nicer full width, IMO.

File: components/Notice.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+interface Props {
+  title: string,
```
Comment (andymsuk) [inline components/Notice.tsx]:
```suggestion
  title: string;
```
Either works but nice to be consistent.

File: pages/[categoryId]/index.tsx (hunk 2 above)
Comment (vanitabarrett) [reply pages/[categoryId]/index.tsx:23]: Good suggestion, thanks! Have moved.

File: styles/components/_notice.scss (hunk 3 above)
Comment (vanitabarrett) [reply styles/components/_notice.scss:3]: Moved to bottom of the page and kept at the original widths.

**Review by andymsuk — APPROVED** [review]

**Pull request merged**